### ISSUE TITLE
[Thu] Train PyTorch biosphere profiler (33 -> 128 -> 64 -> 17)

### ISSUE THREAD

**Issue by zajalist** [zajalist/GeoForge#40]
Train PyTorch neural network:
Input: 33 features
Hidden: 128, 64 neurons (ReLU)
Output: 17 biome classes (softmax)
~3 hours runtime on RTX 5070 (background task)
